src/app/app.component.html

```diff
@@ -226,21 +226,49 @@
           </form>
         </div>
 
-        <div class="footer uk-margin-remove-top" uk-grid>
-          <div class="uk-width-1-1 uk-text-center">
-            <a class="footer-link donation-icon" routerLink="/send" [queryParams]="{ to: donationAccount }" routerLinkActive="active" uk-tooltip title="Contribute to this wallet with a custom donation!"><span uk-icon="icon: heart;"></span></a>
-            <a href="https://github.com/Nault/Nault" uk-icon="icon: github-alt" uk-tooltip title="View our GitHub" class="footer-link" target="_blank" rel="noopener noreferrer"></a>
-            <a href="https://twitter.com/TheNanoCenter" uk-icon="icon: twitter" uk-tooltip title="Chat with us on Twitter" class="footer-link" target="_blank" rel="noopener noreferrer"></a>
-            <a href="https://discord.nanocenter.org" uk-icon="icon: users" uk-tooltip title="Join our community on Discord" class="footer-link" target="_blank" rel="noopener noreferrer"></a>
+        <div class="footer footer-actions uk-margin-remove-top uk-flex-middle" uk-grid>
+          <div class="uk-width-1-3 uk-text-center">
+            <a class="support-nault" routerLink="/send" [queryParams]="{ to: donationAccount }" routerLinkActive="active">
+              <div class="button">
+                <a class="footer-link donation-icon"><span uk-icon="icon: heart; ratio: 1.2;"></span></a>
```

Comment:
Don't know if a solid white heart icon looks better, but unfortunately UIkit does not have a solid white one.

Reply:
it does, but only on hover. added that